web/src/components/genderRadio/genderRadio.css

```diff
@@ -0,0 +1,64 @@
+.genderRadio-wrapper {
+  position: relative;
+  min-height: 40px;
+  display: flex;
+  /*background-color: var(--white);*/
```

Comment:
comment left over from debugging?

Reply:
Less debugging more another option for coloring. Surely this info would be better stored in an md file but if the design is approved like this they can simply be removed.